discord/commands/core.py

```diff
@@ -1016,10 +1016,10 @@ def __init__(
         parent: Optional[SlashCommandGroup] = None,
         **kwargs,
     ) -> None:
-        validate_chat_input_name(name)
-        validate_chat_input_description(description)
-        self.name = str(name)
-        self.description = description
+        self.name = name
+        self.description = description or "No description provided"
+        validate_chat_input_name(self.name)
+        validate_chat_input_description(self.description)
```

Comment:
How are these changes related to the topic of this PR?

Reply:
I found the bug while working on groups. I'll move it to another pr